modules/dkan/dkan_datastore/src/Pages.php

```diff
@@ -67,14 +69,9 @@ private function chooseManagerForm($form) {
     }
     $form['datastore_managers_selection'] = array(
       '#type' => 'select',
-      '#title' => t('Choose a Datastore Manager:'),
+      '#title' => t('Change datastore importer:'),
       '#options' => $options,
-    );
-
-    $form['actions'] = array('#type' => 'actions');
-    $form['actions']['submit'] = array(
-      '#type' => 'submit',
-      '#value' => t("Save"),
+      '#default_value' => "\\$class",
```

Comment:
This seems a little strange/hacky - @fmizzell let me know if you think there's a better way to do this.